frontend/src/components/annotation/image_and_video_annotation/annotation_core.vue

```diff
@@ -185,7 +185,6 @@
               in order for the canvas render to detect it -->
 
           <div
-            contenteditable="true"
```

Comment:
Please note that I removed `contenteditable` ([docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable)) from the div element that wraps the main canvas. The reason I did this is that it was preventing hotkey-js from firing hotkey handlers. This happens because hotkey-js has the code (see bellow) that checks `isContentEditable` property to determine if should fire hotkey handler. My other option was to change hotkey-js filter logic, but I decided to remove `contenteditable` property because I didn't see how it can be used in this context. Specifically, `contenteditable` allows editing of the text inside html elements (like `div`s), but it does not look like we edit any text like that in the `div` container for the `canvas` element.  This change makes the overall design simplier. Please let me know if I'm wrong and I'll be happy to just update hotkey-js filter.


```
const target = event.target || event.srcElement;
        const { tagName } = target;
          let flag = true;
          // ignore: isContentEditable === 'true', <input> and <textarea> when readOnly state is false, <select>
          if (
            target.isContentEditable
            || ((tagName === 'INPUT' || tagName === 'TEXTAREA' || tagName === 'SELECT') && !target.readOnly)
          ) {
            flag = ture
          }
```